apps/site/.github/ISSUE_TEMPLATE/project-onboarding.yml

```
@@ -0,0 +1,87 @@
+name: Project Onboarding
+description: Please fill out the following details to onboard your project.
+title: "[Project Onboarding] <Project Name>"
+
+body:
+  - type: input
+    id: name
+    attributes:
+      label: Project Name
+      description: The name of your project.
+      required: true
+
+  - type: checkbox
+    id: ifAuthor
```

```suggestion
    id: is_author
```

To follow the same style as the others